src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java

```diff
@@ -124,6 +125,12 @@
 
     private ReferenceConfidenceVariantContextMerger merger;
 
+    // the INFO field annotation key names to remove
+    private final List<String> infoFieldAnnotationKeyNamesToRemove = new ArrayList<>();
+
+    // INFO Header names that require alt alleles
+    final Set<String> infoHeaderAltAllelesLineNames = new LinkedHashSet<>();
```

Comment:
I would type this as a LinkedHashSet if the order is important.  Otherwise someone will break it in a confusing way later.

Reply:
Done.